docs/architecture/cloud-native/monitoring-azure-kubernetes.md

```diff
@@ -11,26 +11,26 @@ The built-in logging in Kubernetes is primitive. However, there are some great o
 
 ## Elastic Stack
 
-The Elastic Stack is a powerful option for gathering information from a Kubernetes cluster. Kubernetes supports sending logs to an Elasticsearch endpoint, and for the [most part](https://kubernetes.io/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/), all you need to get started is to set the environment variables as shown in Figure 7-5:
+The Elastic Stack is a powerful option for gathering information from a Kubernetes cluster. Kubernetes supports sending logs to an Elasticsearch endpoint, and for the [most part](https://kubernetes.io/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/), all you need to get started is to set the environment variables as shown in Figure 7-8:
 
 ```kubernetes
 KUBE_LOGGING_DESTINATION=elasticsearch
 KUBE_ENABLE_NODE_LOGGING=true
 ```
 
-**Figure 7-5** - Configuration variables for Kubernetes
+**Figure 7-8** - Configuration variables for Kubernetes
 
 This will install Elasticsearch on the cluster and target sending all the cluster logs to it.
 
 ![An example of a Kibana dashboard showing the results of a query against logs ingested from Kubernetes](./media/kibana-dashboard.png)
-**Figure 7-6**. An example of a Kibana dashboard showing the results of a query against logs that are ingested from Kubernetes
+**Figure 7-9**. An example of a Kibana dashboard showing the results of a query against logs that are ingested from Kubernetes
 
 ## Azure Container Monitoring
```

Comment:
You mean Azure Monitor for Containers: https://docs.microsoft.com/en-us/azure/azure-monitor/insights/container-insights-overview 
Can you make this the first option and move the ELK Stack content to the chapter "Logging with Elastic Stack"?

Reply:
Can you also briefly mention about https://prometheus.io/ and the integration the Azure Monitor offers?